packages/db/src/migrations/20250915152638_create_granule_groups_table_and_index.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+export const up = async (knex: Knex): Promise<void> => {
+  await knex.schema.createTable('granule_groups', (table) => {
+    table
+      .integer('granule_cumulus_id')
+      .references('cumulus_id')
+      .inTable('granules')
+      .notNullable()
+      .onDelete('CASCADE')
+      .primary();
+    table
+      .specificType('state', 'char(1)')
+      .comment('Granule active state')
+      .notNullable();
+    table
+      .integer('group_id')
+      .comment('Granule duplicate group id')
+      .notNullable();
+    table
+      .timestamps(false, true);
+    table
+      .unique(['granule_cumulus_id']);
```

Comment:
This `unique` constraint should be removed.  A primary key is already: NOT NULL, UNIQUE, Indexed